app/src/main/res/drawable/edit_text_background_border.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="3dp" />
```

Comment:
Check mocks -> I think it should be `4dp`

Reply:
okay